### ISSUE TITLE
v7 client pause (core shutdown) looks like a failed WU to HFM due to the core return value.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [harlam357/hfm-net#271]
```
What steps will reproduce the problem?

1. Pause a v7 client slot

What is the expected output? What do you see instead?

- Pausing a client should not be noted as a WU failure.  However, the cores do 
not generally exit cleanly like they did in v6 and return values that are 
associated with WU failures in v6.

- I'm going to have to override the logging base code with specific code for 
v7.  May have to look for a message that notes the slot is pausing and then 
basically ignore the core return code.


```

Original issue reported on code.google.com by `harlam357` on 28 Mar 2012 at 3:40

**Comment by harlam357** [comment]
Closed - HFM.Log API v2 in 0.9.7 addresses this issue.
